src/redux/catagories/catagories.js

```diff
@@ -0,0 +1,19 @@
+const STATUS = 'CHECKING_STATUS';
```

Comment:
- [ ] Same issue here. It would be best to use the [duck pattern](https://github.com/erikras/ducks-modular-redux) to write this action 👍